plugins/protocolbuffers/java/v24.4/buf.plugin.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
+name: buf.build/protocolbuffers/java
+plugin_version: v24.4
+source_url: https://github.com/protocolbuffers/protobuf
+description: Base types for Java. Generates message and enum types.
+output_languages:
+  - java
+spdx_license_id: BSD-3-Clause
+license_url: https://github.com/protocolbuffers/protobuf/blob/v24.4/LICENSE
+registry:
+  maven:
+    deps:
+      - com.google.protobuf:protobuf-java:3.24.4
+    additional_runtimes:
+      - name: lite
+        deps:
+          - com.google.protobuf:protobuf-javalite:3.24.4
+          - build.buf:protobuf-javalite:3.24.4
```

Comment:
I forgot we have a dependency on this. And we should avoid publishing java and co. before the javalite version is available which is in a separate repository

Reply:
This has been released (thank you), 😌 

https://repo1.maven.org/maven2/build/buf/protobuf-javalite/